### ISSUE TITLE
Services for Application Groups

### ISSUE THREAD

**Issue by Emi-Emi-Emi** [tnodir/fort#135]
Currently I have to create an empty Application Group, and select every Service to use that application group so I can easily turn many services on and off at once.
It would be nice if Services path `\svchost\` worked in the Applications Groups Block/Allow list, to do the same but 'better', while clearing Programs' list from services I don't need there like I do with normal executables.

Don't know if it can be done, but requesting it in case.

**Comment by tnodir** [comment]
It should already work. Let me double check..

**Comment by Emi-Emi-Emi** [comment]
Well, I tested it with a total new Fort data earlier to make sure and wasn't working, because I just added the same path of whatever service that got blocked, and I added it to the block side and it was still getting blocked again and appearing in Programs.

Now I re-tested it and it works, I even went ahead and tested wildcards and they work.

I checked things 10 times to see if it wasn't working, guess I am rust at testing stuff 👍the mysteries of technology.

**Comment by tnodir** [comment]
> I just added the same path of whatever service that got blocked, and I added it to the block side and it was still getting blocked again and appearing in Programs.

Yes, it's a bug I mentioned above.

**Comment by Emi-Emi-Emi** [comment]
Oh yes, but I thought you said it was by design not a bug, I remember we had the discussion here https://github.com/tnodir/fort/issues/11  

But what I meant is I kept deleting from Programs, so only used it for copying and pasting the service path in the Application Programs and it kept re-appearing as not being blocked by Application Groups and by normal Programs.

I used a total new Data to avoid conflicts, like I just noticed I had two programs named Main, because because I was testing the bug with the Exclude/Include All. 

So I made a request based on something not working on my side, it is good the feature works! Thanks for best firewall.
